front/src/Phaser/Game/GameScene.ts

```diff
@@ -120,14 +134,16 @@ export class GameScene extends Phaser.Scene implements GameSceneInterface{
         //notify game manager can to create currentUser in map
         this.GameManager.createCurrentPlayer();
 
-
         //initialise camera
         this.initCamera();
 
 
         // Let's generate the circle for the group delimiter
 
         this.circleTexture = this.textures.createCanvas('circleSprite', 96, 96);
+        if(!this.circleTexture || this.circleTexture.context){
```

Comment:
This is breaking the display of the circle, because `this.circleTexture.context` should always evaluate to true.
Why did you need this? Is something breaking?

Reply:
Yes the changes have break circle feature. I will check and try to fix it.